core/src/sigverify_stage.rs

```diff
@@ -171,9 +174,12 @@ impl SigVerifierStats {
             ("total_packets", self.total_packets, i64),
             ("total_dedup", self.total_dedup, i64),
             ("total_excess_fail", self.total_excess_fail, i64),
-            ("total_shrink_time", self.total_shrink_time, i64),
-            ("total_shrinks", self.total_shrinks, i64),
             ("total_valid_packets", self.total_valid_packets, i64),
+            ("total_shrinks", self.total_shrinks, i64),
```

Comment:
can we add _us here to make it clear what unit these are?

Reply:
I could add it for the new metrics. I assumed changing a metric field name would cause issues, so would keep `total_shrink_time` as it was. Does that sound right?

Reply:
should be fine to change it, it's relatively new I don't think there is any dashboard with it at the moment.

Reply:
done